addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -201,12 +201,41 @@ function ActivityFactory({ Entity }) {
             });
         }
 
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsCurrentPartnerAssignee() {
+            if (
+                !this.assignee ||
+                !this.assignee.partner ||
+                !this.env.messaging.currentPartner
```

Comment:
Since you now have the messaging field, it would be better to use `this.messaging` (general rule: try to not access singleton entities from env/messaging if you have a direct reference to them already).